cpu/cc3200/include/cc3200_prcm.h

```diff
@@ -35,12 +35,13 @@ extern "C" {
  * @brief convert x usecs to cpu cycles
  *
  */
-#define USEC_TO_CPU_CYCLES(x) (80 * x) / 3
+#define USEC_TO_CPU_CYCLES(x) ((80000000U / 1000000) * x)
```

Comment:
Don't you want to write `CLOCK_CORECLOCK ` instead of `80000000U`? ;)

Reply:
That was quick 😄
I am just having trouble importing `periph_conf.h` here... Need to check why that is.
I moved the division by 3 to USEC_DELAY since it is not actually required for converting to usec. ROM_UtilsDelay takes 3 cycles so I divide there so it is a little clearer